README.md

```diff
@@ -11,6 +11,8 @@ Material Components for the web is the successor to [Material Design Lite](https
 
 MDC-Web strives to seamlessly incorporate into a wider range of usage contexts, from simple static websites to complex, JavaScript-heavy applications to hybrid client/server rendering systems. In short, whether you're already heavily invested in another framework or not, it should be easy to incorporate Material Components into your site in a lightweight, idiomatic fashion.
 
+> **Note:** MDC-Web is intended to be used for other developers to execute Material Design properly. MDC-Web delegates higher level capabilities to other JS frameworks and libraries. An example is that MDC-Web neither use web components nor renders the DOM of its components. By doing low-level operations, MDC-Web its able to accomplish its goals.
```

Comment:
I still have concerns about the message that we are trying to communicate. Before I iterate with your feedback, I would like to make sure we are on the same page. Initially, I thought that the main goal for this PR is to clarify that material components for the web does not offer "Web components" but a "bootstrap-y component" solution. The UX of these solutions are drastically different from each other, e.g. one offers encapsulation and the other one doesn't. Another analogy is the difference between a black box (Web components) vs a white box (bootstrap). Implicitly this adds new different audiences to the table:

1. Users that want to use these components by just thinking of black boxes. This is a higher level operation from a user's perspective, just plug and play. More suited to prototypers, first time learners, etc.

2. Users that want to have total control of components by working with white boxes (Low-level operations). This is more suited to developers that want to customize and/or build wrappers on their own frameworks.

Thoughts?